images/warp10/s6/cont-init.d/80-configure-exporter

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/with-contenv sh
+echo "configuring exporter"
+pwd
+ls
+mv /warp10-sensision-exporter-master/ /run/s6/services/sensision-exporter/
```

Comment:
this seems like a waste of a cont-init.d file, but s6 creates directories after the exporter build. Should be a better way to do this